demisto_sdk/commands/generate_test_playbook/test_playbook_generator.py

```diff
@@ -276,10 +276,32 @@ def __init__(self, input: str, output: str, name: str, file_type: str, no_output
                  verbose: bool = False, use_all_brands: bool = False):
         self.integration_yml_path = input
         self.output = output
+
+        generated_test_playbook_file_name = f'playbook-{name}_Test.yml'
+
         if output:
-            self.test_playbook_yml_path = os.path.join(output, name + '.yml')
+            """ if an output folder path is provided, save it there"""
+            output_path = Path(output)
+            if output_path.is_dir():
+                self.test_playbook_yml_path = str(output_path / generated_test_playbook_file_name)
+            else:
+                """ if a specific destination path is specified for the playbook, use it"""
+                self.test_playbook_yml_path = output
```

Comment:
This is for handling case where output is a path to a file name?

Reply:
indeed.
I thought of checking `if self.test_playbook_yml_path.endswith('yml')` but wasn't sure about it. wdyt?

Reply:
I think checking it with a good print msg in case the file does not end with YML could be nice. Will prevent errors from starting developers in the future

Reply:
so if the destination doesn't end with `yml`, should we add the extension or leave it as is?
(i'll add a `print` anyway)

Reply:
We should stop the generation and return a message to the user that YML file was expected but file is not YML etc